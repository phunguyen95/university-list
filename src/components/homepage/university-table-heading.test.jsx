import { render } from "@testing-library/react";
import UniversityTableHeading from "./university-table-heading";


test("renders UniversityTableHeading  view without crashing", () => {

  const { asFragment } = render(<UniversityTableHeading />);

  expect(asFragment()).toMatchSnapshot();
});
