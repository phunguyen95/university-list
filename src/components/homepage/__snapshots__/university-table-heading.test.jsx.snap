// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders UniversityTableHeading  view without crashing 1`] = `
<DocumentFragment>
  No.NameCountryWebsite
</DocumentFragment>
`;
